### PR TITLE
chore(tech): 🤖 added commitizen for conventional commits

### PR DESCRIPTION
**Pull request by airaamane**
added Commitizen to this repo in order to have a more consistent commit style across contributors.

### CODE REVIEW

**Comment by chihab** [comment]
Thanks for the contrib @airaamane, nice idea!

File: package.json [hunk 1]
```diff
@@ -31,6 +32,9 @@
     "@types/jasmine": "~3.6.0",
     "@types/node": "^12.11.1",
     "codelyzer": "^6.0.0",
+    "commitizen": "^4.2.3",
+    "git-cz": "^4.7.6",
+    "husky": "^5.1.3",
```
Comment (chihab) [inline package.json]: I think you've added husky to enforce cz on each commit but the hook is not configured.

File: README.md [hunk 2]
```diff
@@ -9,6 +11,7 @@ This is the official `Angular in darija` landing page containing all video sessi
 - Run `npm start` 
 - Tests `npm run test`  
 - End to end tests `npm run e2e`
+- Conventional git commit `npm run cz` (please use instead of `git commit` whenever possible)
```
Comment (chihab) [inline README.md:14]: If we use husky (see below) we'll enforce using cz on each commit.

File: changelog.config.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+module.exports = {
```
Comment (chihab) [inline changelog.config.js]: I would suggest to install the cz adapter (as a npm devdep) and reference it in cz conf.

File: README.md (hunk 2 above)
Comment (airaamane) [reply README.md:14]: my bad, there should be a hook for that, will add and republish

File: changelog.config.js (hunk 3 above)
Comment (airaamane) [reply changelog.config.js]: can we still have granular control like we have here? git-cz is the adapter i'm using

Comment (chihab) [reply changelog.config.js]: I think we should use the default config from the adapter without having it in our workspace, then maybe override it when we'll need to.

**Pull request closed without merging**